jiant/preprocess.py

```diff
@@ -785,11 +870,15 @@ def __init__(self, args):
         else:
             boundary_token_fn = utils.apply_standard_boundary_tokens
 
-        self.boundary_token_fn = boundary_token_fn
+        # TODO: consider the max_tokens from a model-specific config instead of args.max_seq_len?
```

Comment:
probably should be something like `min(args.max_seq_len, model.max_pos)` (assuming `args.max_seq_len` is set, and defaulting to `model.max_pos` if not)

Reply:
👍, done w/ 65912cc